packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/RecordHistoryEvents/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { Grid, Header } from 'semantic-ui-react'
+
+import { withI18n } from 'coreModules/i18n/higherOrderComponents'
+
+import EventRow from './EventRow'
+
+const mapStateToProps = (state, { formValueSelector, name }) => {
+  return {
+    recordHistoryEvents: formValueSelector(state, name),
+  }
+}
+
+const propTypes = {
+  i18n: PropTypes.shape({
+    moduleTranslate: PropTypes.func.isRequired,
+  }).isRequired,
+  recordHistoryEvents: PropTypes.arrayOf(
+    PropTypes.shape({
+      action: PropTypes.string,
+      srcCreatedAt: PropTypes.string,
+      srcUpdatedAt: PropTypes.string,
+      userName: PropTypes.string,
+    })
+  ),
+}
+const defaultProps = {
+  recordHistoryEvents: [],
+}
+
+class RecordHistoryEvents extends Component {
+  render() {
+    const { i18n: { moduleTranslate }, recordHistoryEvents } = this.props
+
+    const createdEvent = recordHistoryEvents.find(
+      ({ action }) => action === 'create'
+    )
+    const lastModifiedEvent = recordHistoryEvents.find(
+      ({ action }) => action === 'update'
+    )
+
+    const hasEvents = recordHistoryEvents.length > 0
+    return (
+      <Grid.Column width={16}>
+        {hasEvents && (
```

Comment:
if you use TranslatedHeader, as commented above, you will have a column around the header. then this column should not be needed anymore, so it can be a React.Fragment instead. this is also good because as it is now, we would render a column even if there are no events, but that is unnecessary